OntologyMapper/test/OntologyMappingManagerTests.cs

```diff
@@ -115,19 +115,55 @@ public void TryGetPropertyProjection_ReturnsTrue_WhenFound()
 
             var ontologyMappingManager = new OntologyMappingManager(mockOntologyLoader.Object);
 
-            var outputDtmiFilter = "*";
+            var outputDtmi = "dtmi:org:w3id:rec:Space;1";
             var outputPropertyName = "externalIds";
             var inputPropertyName = "deviceKey";
 
-            var result = ontologyMappingManager.TryGetPropertyProjection(outputDtmiFilter, outputPropertyName, out var propertyProjection);
+            var result = ontologyMappingManager.TryGetPropertyProjection(outputDtmi, outputPropertyName, out var propertyProjection);
 
             Assert.True(result);
             Assert.NotNull(propertyProjection);
-            Assert.Equal(outputDtmiFilter, propertyProjection.OutputDtmiFilter);
             Assert.Equal(outputPropertyName, propertyProjection.OutputPropertyName);
             Assert.Equal(inputPropertyName, propertyProjection.InputPropertyNames[0]);
         }
 
+        [Fact]
+        public void TryGetPropertyProjection_ReturnsTrue_WhenRegexMatchFound()
```

Comment:
Should there also be a test for an overlapping model that exists that fits the pattern? For example:  `dtmi:org:w3id:rec:BoatMotor;1`